gitensite/apps/content/templates/license.html

```diff
@@ -0,0 +1,31 @@
+{% extends 'frame.html' %}
+{% load fontawesome %}
+{% load staticfiles %}
+
+{% block title %} License -- GITenberg - Making beautiful books in the public domain {% endblock title %}
+
+{% block body_content %}
+<div class='row'>
+  <div class="large-offset-2 large-8 small-12 columns">
+    <h1>Copyright and Licensing</h1>
+    <p>
+      All work by the GITenberg project is released under an open license.
+    </p>
+    <p>
+      Our books in the public domain, and we claim no further copyright on them.
+    </p>
+    <p>
+      Some of our books contain a cover image from the 
+      <a href="http://shop.thecreativeactionnetwork.com/collections/recovering-the-classics">Recovering the Classics</a> project from Creative Action Network.
+      These covers are not in the public domain but are graciously available for use under the 
```

Comment:
the NC license is partly in conflict with the "Our books are free to use by anyone for any purpose".